cas-server-support-osf/src/main/java/org/pac4j/oauth/client/OrcidClient.java

```diff
@@ -120,12 +120,22 @@ protected String getProfileUrl(final Token accessToken) {
     @Override
     protected void internalInit() {
         super.internalInit();
-        this.service = new ProxyOAuth20ServiceImpl(new OrcidApi20(this.getMember()), new OAuthConfig(this.key,
-                this.secret,
-                this.callbackUrl,
-                SignatureType.Header,
-                this.getScope(),
-                null), this.connectTimeout, this.readTimeout, this.proxyHost, this.proxyPort, false, true);
+        this.service = new ProxyOAuth20ServiceImpl(
+                new OrcidApi20(),
```

Comment:
The only functional change here is `new OrcidApi20(this.getMember())` -->> `new OrcidApi20()`. `member` is no longer needed for the API endpoint.